app/views/users/edit.html.erb

```diff
@@ -1,3 +1,16 @@
+<!--
+<form action="/users" method="post" accept-charset="UTF-8">
+Username: <input type="text" name="username"><br> <br>
+
+Email: <input type="email" name="email"><br><br>
+
+Password: <input type="password" name="password"><br>
+<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="submit" value="Submit">
+</form>
+-->
+
+
```

Comment:
Hi, team!

You have not tried out form_tag in this project. Kindly look at this [material](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/FormTagHelper.html) as a reference. Also, read this [material](https://guides.rubyonrails.org/v3.0/form_helpers.html). Scroll through the web pages carefully to see the topic on form_tag. :slightly_smiling_face: